sdk-java/src/main/java/ly/count/sdk/java/Config.java

```diff
@@ -865,13 +866,34 @@ public Config setSendUpdateEachSeconds(int sendUpdateEachSeconds) {
         return this;
     }
 
+    /**
+     * Set maximum amount of time in seconds between two update requests to the server
+     * reporting session duration and other parameters if any added between update requests.
+     *
+     * Update request is also sent when number of unsent events reached {@link #setEventQueueSizeToSend(int)}.
+     *
+     * @param delay max time interval between two update requests, set to 0 to disable update requests based on time.
+     * @return {@code this} instance for method chaining
+     */
+    public Config setUpdateSessionTimerDelay(int delay) {
+        if (sendUpdateEachSeconds < 0) {
+            if (configLog != null) {
+                configLog.e("[ConfigCore] sendUpdateEachSeconds cannot be negative");
```

Comment:
this seems to be the old log, what do you think?

Reply:
Should I change all ConfigCore occurences? @ArtursKadikis

Reply:
yup, pls do

Reply:
done